src/bentoml/types.py

```diff
@@ -1,7 +1,4 @@
-from typing import TYPE_CHECKING
+from ._internal.models.model import ModelSignature
+from ._internal.models.model import ModelSignatureDict
```

Comment:
actually this won't work, maybe we need to import TypedDict from `typing_extensions` and just make ModelSignatureDict a type that you can just use to initialize with